client-hc/pom.xml

```diff
@@ -54,19 +54,4 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <configuration>
-                    <compilerArgument>-XDignore.symbol.file</compilerArgument>
```

Comment:
Does anyone know what this `ignore.symbol.file` was meant to do and if it is still needed?

Reply:
Looks like the intention was access to internal JRE classes.